src/components/Footer.js

```diff
@@ -1,12 +1,24 @@
 import React from 'react';
+import { NavLink } from "react-router-dom";
 
 const Footer = () => {
     return (
-        <>
-        
-        </>
-    )
-
-}
-
+        <footer className="footer">
+          <div className="container">
+            <div className="row">
+              <div className="col-md-4">
+                <h1>The TrapHouse</h1>
```

Comment:
The h1 tag is used for a header and is odd, semantically, to have in the footer. Can you change it to something else? A paragraph tag would work.

Reply:
Swapped h1 for h2 for style optimization.